src/libmscore/lyrics.h

```diff
@@ -82,7 +82,7 @@ class Lyrics final : public TextBase
     QString subtypeName() const override { return QObject::tr("Verse %1").arg(_no + 1); }
     void setNo(int n) { _no = n; }
     int no() const { return _no; }
-    bool isEven() const { return _no % 1; }
+    bool isEven() const { return _no % 2; }
```

Comment:
I think it should be `!(_no % 2)`

Reply:
oh, yes, or ```_no&1```, it's my error too

Reply:
You can use `!(_no & 1)` too, it's faster, but the speed probably doesn't matter...